src/iuse.cpp

```diff
@@ -5676,21 +5681,22 @@ void item::extended_photo_def::serialize( JsonOut &jsout ) const
     jsout.end_object();
 }
 
-std::optional<int> iuse::einktabletpc( Character *p, item *it, bool t, const tripoint &pos )
+std::optional<int> iuse::epic_music( Character *p, item *it, const tripoint &pos )
 {
-    if( t ) {
-        if( !it->get_var( "EIPC_MUSIC_ON" ).empty() &&
-            it->ammo_sufficient( p ) ) {
-            if( calendar::once_every( 5_minutes ) ) {
-                it->ammo_consume( 1, p->pos(), p );
-            }
+    if( !it->get_var( "EIPC_MUSIC_ON" ).empty() &&
```

Comment:
```suggestion
    if( !it->get_var( "EPIC_MUSIC_ON" ).empty() &&
```
Typo or am I messing things up?

Reply:
That typo already existed before me. AFAIK working as intended.

https://github.com/CleverRaven/Cataclysm-DDA/blob/383e125594ba442751c94176c948d6501398a274/src/iuse.cpp#L5682